src/js/module/test/comment.test.js

```diff
@@ -0,0 +1,15 @@
+import generateCommentHTML from '../popup/commentshow.js';
+
+test('generateCommentHTML should generate correct comment HTML', () => {
+  const comment = {
+    creation_date: '2023-06-28',
+    username: 'JohnDoe',
+    comment: 'This is a test comment.',
+  };
+
+  const expectedHTML = `<p>${comment.creation_date} ${comment.username}: ${comment.comment}</p>`;
+
+  const generatedHTML = generateCommentHTML(comment);
+
+  expect(generatedHTML.trim()).toBe(expectedHTML.trim());
```

Comment:
- [ ] This comment popup test is failing, and for the comment test, you need to test the number of comments, not the HTML as requested in the project requirements. Please make sure to test the correct values and the test should pass without any issues.

![image](https://github.com/YoutLone/Capstone-Js/assets/83803414/0fe668ae-31cc-485d-b49f-7903a331b587)